datamade/la-metro-councilmatic
# Include appearing on agenda in board report history, update columns

## Overview

This PR:

- Adds an `actions_and_agendas` property to the `LAMetroBill` model that includes information about events and agendas, their times, descriptions, and related organizations
- updates the `legislation.html` template (for the `LAMetroBill` model) with the data from this property

This PR pulls the relevant commits from #450, which it supersedes. (Let's squash and merge when it comes in, to get rid of the mysterious duplicate commits.)

### Checklist


## Testing Instructions

* Navigate to a piece of legislation
* Hard refresh the page
* Confirm link goes to appropriate event agenda
* Confirm event status is correct
* Repeat with a few different pages

Handles #348
